docs/guides/using-store-outside-react.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Using store outside React
+description: Using store outside React
+nav: 4.98
+keywords: state, outside, react
+published: false
+---
+
+Jotai's state resides in React, but sometimes it would be nice
+to interact with the world outside React.
+
+## createStore
+
+[`createStore`](../core/store) provides a store interface that can be used to store your atoms. Using the store, you can access and mutate the state of your stored atoms from outside React.
```

Comment:
We need the extension for browsing in GitHub.
```suggestion
[`createStore`](../core/store.mdx) provides a store interface that can be used to store your atoms. Using the store, you can access and mutate the state of your stored atoms from outside React.
```

Reply:
<img width="778" alt="image" src="https://github.com/pmndrs/jotai/assets/60546840/da842001-1451-490c-85d7-a822de7550ce">

Done
added `.mdx#createstore`

Reply:
> It's bare minimum, but a good start.

Is there anything else you would like me to add here?